FirebaseAuth/Sources/User/FIRUser.m

```diff
@@ -374,7 +374,9 @@ - (nullable instancetype)initWithCoder:(NSCoder *)aDecoder {
     _phoneNumber = phoneNumber;
     _metadata = metadata ?: [[FIRUserMetadata alloc] initWithCreationDate:nil lastSignInDate:nil];
     _tenantID = tenantID;
-    _requestConfiguration = [[FIRAuthRequestConfiguration alloc] initWithAPIKey:APIKey appID:appID];
+    _requestConfiguration = [[FIRAuthRequestConfiguration alloc] initWithAPIKey:APIKey
+                                                                          appID:appID
+                                                                heartbeatLogger:nil];
```

Comment:
Thanks for the context, it may be worth mentioning in a comment that `heartbeatLogger` will be set later via a property update so it's clear. No need to mention the `NSCoding` implementation details though.

Reply:
Done in [c9d719a](https://github.com/firebase/firebase-ios-sdk/pull/9556/commits/c9d719a75950c04a5f00f9bc77811e2b66028463)